RosaServer/api.cpp

```diff
@@ -1626,6 +1626,10 @@ Item* Item::getParentItem() const {
 
 RigidBody* Item::getRigidBody() const { return &Engine::bodies[bodyID]; }
 
+Item* Item::getChildItem(int idx) const {

```

Comment:
Needs to be unsigned. What if idx is -5?

Reply:
Done.